setup.py

```diff
@@ -1,7 +1,7 @@
 from setuptools import setup
 
 setup(name='sdcclient',
-      version='0.10.0-dev',
+      version='0.10.1-dev',
```

Comment:
Out of curiosity, how do you "use" this change?

Reply:
CLI has a requirement on `sdcclient`. I guess that's the reason for the update